lib/cocoapods/installer/xcode/pods_project_generator.rb

```diff
@@ -142,10 +142,10 @@ def install_aggregate_targets(project, aggregate_targets)
         #
         def integrate_targets(pod_target_installation_results)
           pod_installations_to_integrate = pod_target_installation_results.values.select do |pod_target_installation_result|
-            pod_target = pod_target_installation_result.target
             !pod_target_installation_result.test_native_targets.empty? ||
                 !pod_target_installation_result.app_native_targets.empty? ||
-                pod_target.contains_script_phases?
+                pod_target_installation_result.target.contains_script_phases? ||
+              pod_target_installation_result.target.framework_paths.values.flatten.any? { |paths| !paths.dsym_path.nil? }
```

Comment:
why not keep the `pod_target` variable and use it in these two lines?

Reply:
sure

Reply:
done!